megamek/src/megamek/common/Tank.java

```diff
@@ -1669,6 +1671,7 @@ public double getCost(CalculationReport calcReport, boolean ignoreAmmo) {
 
     @Override
     public double getPriceMultiplier() {
+        // TechManual, 1st printing, p285 "Final Unit Cost Formulas Table".
```

Comment:
I checked against TechManual 6th printing, page 285 "Final Unit Cost Formulas" and it matches 1:1